config/custom-dispatching/util/batch-update.js

```diff
@@ -0,0 +1,134 @@
+async function parallelisedBatchedUpdate(
```

Comment:
Content of this file is copied over from the `mandataris-besluiten-consumer-app`